client/components/Coverages/CoverageEditor/CoverageForm.tsx

```diff
@@ -310,6 +311,69 @@ export class CoverageFormComponent extends React.Component<IProps, IState> {
         }
     }
 
+    onAddToWorkflowChange() {
+        const {vocabulary} = superdeskApi.entities;
+        const {gettext} = superdeskApi.localization;
+        const coverageStatuses = vocabulary
+            .getAll()
+            .get('newscoveragestatus').items as Array<IPlanningNewsCoverageStatus>;
+        const updatedCoverage = planningUtils.addCoverageToWorkflow(this.props.value, coverageStatuses);
+        const filteredCoverages = this.props.coverages.filter((x) => x.coverage_id !== updatedCoverage.coverage_id);
```

Comment:
Choose a better variable name here that would better describe the value it holds and what you're trying to achieve with it. I see that you're filtering it here, spreading it there, but I don't get the entire picture of what you're doing.

After implementing actual toggling, would this function also be called when removing item from workflow? It would need to be renamed then.